rfcs/cmt_input.md

```diff
@@ -0,0 +1,52 @@
+# Cmt files - both as input and output
+
+This RFC asks for two separate additions to the compiler front end. They are
+technically independent, but mostly useful together. So I decided to bundled
+them.
+
+The first request is to type check compilation units and produce `.cmt` files:
+
+```
+$ ocamlc -c foo.ml -o foo.cmt
```

Comment:
I think it should be possible to do this without having to put the detection in the name given to `-o`, e.g. something like `ocamlc -c foo.ml -output-cmt`.

Reply:
Seems alright to me. I have to say that I prefer a command line without so many options to remember, but I will never run this command manually in practice.